README.md

```diff
@@ -16,7 +16,33 @@ TODO
 
 ### Interactions on GitHub
 
-TODO
+**Issue Tracker**
+
+- Label with accurate and descriptive issue type (use available labels)
+
+- Assign yourself and teammates to corresponding issue
+
+- Briefly and accurately describe the issue at hand and the possible solution
+
+- Address issues quickly and don't let them sit longer than 1 week
+
+- Track them through the project board
+
+**Pull Requests**
```

Comment:
A possible addition would be adding a req for how many reviewers need to approve before merging. (2?)